.azure/test-linux.yml

```diff
@@ -52,6 +56,10 @@ jobs:
           path: .stestr
         displayName: "Cache stestr"
 
+      - ${{ if eq(parameters.installFromSdist, true) }}:
+        - bash: |
+            rustup default 1.56
```

Comment:
The parameter test here doesn't match the new parameter name you put in - maybe a typo?  Also, could we put the magic MSRV version number in the `azure-pipelines.yml` file (along with the Python versions), and pass that down to the `test-linux.yml` script, rather than hardcoding it here?  It just keeps all the version numbers in one place.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8196/commits/c86a77cd886b1eb61118d9d2c1e106662df96b69